docs/apps/components/sub-form.md

```diff
@@ -0,0 +1,148 @@
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+    4. Before using **this.enable(fields)**, make sure the component is **disabled**. This can be done using **this.disable(fields)** or by **enabling** the **Disabled checkbox** in the Component Attribute panel.
+
+#### Steps to use the methods for the Page Builder components
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/subform3.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to use the methods for the Page Builder components
+          * ```
+            this.addClassName('table_n7rs950l', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('table_n7rs950l', 'abc')
+        ```
+
+    3. `this.hide(fields)`
+        ```js
+        var fields= ['table_n7rs950l']
+        this.hide(fields)
+        ```
+    
+    4. `this.show(fields)`
+        ```js
+        var fields= ['table_n7rs950l']
+        this.show(fields)
+        ```
+    
+    5. `this.disable(['fields'])`
+        ```js
+        this.disable(['table_n7rs950l'])
+        ```
+    
+    6. `this.enable(['fields'])`
+        ```js
+        this.enable(['table_n7rs950l'])
+        ```
+
+    7.  `this.getComponent('component_ID')`
+        ```js
+        var subname = this.getComponent('table_n7rs950l');
+        console.log('getComponent', subname);
+        ```
+
+!!! info Add multiple classes
+    Use the following syntax to add multiple classes to a component:
+    ```js
+    this.addClassName('componentID', 'class1', 'class2')
+    ```
+
+## Config
+
+| **Name**|**Description**|**Icon**|
+|---------|---------------|--------|
+|**ID**| The sub-form component's unique identifier|<img src= "/apps/components/img/input_id.png">|
+|**Label Width**|Width of the label associated with an input field. It determines the horizontal space occupied by the label text|<img src= "/apps/components/img/input_labelwidth1.png">|
+|**Hide Label**| Hides the label on the form|<img src= "/apps/components/img/input_hidelabel.png">|
```

Comment:
This hide label option is not available on the UI

Reply:
removed